lib/commands/command-info.bash

```diff
@@ -2,9 +2,21 @@
 # shellcheck source=lib/functions/plugins.bash
 . "$(dirname "$(dirname "$0")")/lib/functions/plugins.bash"
 
+get_shell() {
+  if [[ -z $FISH_VERSION ]]; then
+    echo "fish"
```

Comment:
The current code would print something like `fish, version 3.3.1`, whereas this only prints `fish`. I think both version and type are useful.

Reply:
sure thing, i've updated the output in [91527a3](https://github.com/asdf-vm/asdf/pull/1217/commits/91527a3df04b44a69df9de8a9789920ef42ecb0b)